src/components/LibraryInfo/index.js

```diff
@@ -12,15 +12,17 @@ governing permissions and limitations under the License.
 
 import libraryVersion from "../../constants/libraryVersion";
 
-const createLibraryInfo = () => {
+const createLibraryInfo = tools => {
   return {
     commands: {
       getLibraryInfo: {
         run: () => {
           return {
             libraryInfo: {
               version: libraryVersion
-            }
+            },
+            configInfo: tools.config,
+            commandInfo: tools.componentRegistry.getCommandNames()
```

Comment:
"configure" is a command name that is not in the componentRegistry, but is a core command.